packages/components/psammead-idt1-include/src/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,21 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Idt1Include should render with require js in head 1`] = `
+<html>
+  <head>
+    <script
+      async="true"
+      onload="function handleOnLoad() {
+    /* istanbul ignore next */
+    cov_2ibc2ghh9b.f[1]++;
+    cov_2ibc2ghh9b.s[3]++;
+    setIsRequireLoaded(true);
+  }"
+      src="https://news.files.bbci.co.uk/include/vjassets/js/vendor/require-2.1.20b.min.js"
+    />
+  </head>
+  <body>
+    <div />
+  </body>
+</html>
```

Comment:
Initial snapshot showing require is in the head and the div is empty because the include has yet to be inserted